connector_prestashop/delivery.py

```diff
@@ -1,32 +1,11 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Authors: Guewen Baconnier, Sébastien Beau, David Béal
-#    Copyright (C) 2010 BEAU Sébastien
-#    Copyright 2011-2013 Camptocamp SA
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
As the previous work was too much mixed in the authors thing, I have preferred to let all files without copyright for now instead of putting all the possible authors in all the files. The manifest gathers all the authors I have found.